src/CMakeLists.txt

```diff
@@ -134,8 +134,6 @@ if(ENABLE_MULTIO)
    list(APPEND sources_Fortran ${src_home}/ifs_interface/iom.F90)
 endif()
 
-file(GLOB sources_C ${src_home}/*.c)
```

Comment:
Was this the only c code? I seem to recall some fesom fileserver code by Jan Hegewald that is in c. Though I think, we don't use it by default.

Reply:
Not sure what you mean about oce_ale. I removed the calls an logic surrounding the PARMS solver, but maybe I got carried away and removed too much? 
About the other C code, I will check again!

Reply:
The async_threads you mean? those are in CPP and in their own subdirectory.